app/views/dashboard/index.html.slim

```diff
@@ -56,15 +62,15 @@ h1 Cluster Status
           thead
             tr
               th
-                | Id
-              th.text-center
+                | &nbsp;
+              th
                 | Status
               th
-                | Hostname
+                | ID
               th
-                | Role
+                | Hostname
               th.text-center
```

Comment:
You removed `Master` column here. Also, only master column needs `.text-center`, you can remove from this one.